src/test/java/com/endava/endabank/utils/TestUtils.java

```diff
@@ -0,0 +1,197 @@
+package com.endava.endabank.utils;
+
+import com.endava.endabank.constants.Strings;
+import com.endava.endabank.dto.user.UpdatePasswordDto;
+import com.endava.endabank.dto.user.UserDetailsDto;
+import com.endava.endabank.dto.user.UserPrincipalSecurity;
+import com.endava.endabank.dto.user.UserRegisterDto;
+import com.endava.endabank.dto.user.UserRegisterGetDto;
+import com.endava.endabank.dto.user.UserToApproveAccountDto;
+import com.endava.endabank.model.ForgotUserPasswordToken;
+import com.endava.endabank.model.IdentifierType;
+import com.endava.endabank.model.Permission;
+import com.endava.endabank.model.Role;
+import com.endava.endabank.model.User;
+import com.endava.endabank.security.UserAuthentication;
+import com.sendgrid.Response;
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import org.modelmapper.ModelMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class TestUtils {
+    public static final String SECRET_DUMMY = "ZHVtbXkgdmFsdWUK";
+
+    public static User getUserAdmin() {
+        Role adminRole = TestUtils.adminRole();
+        IdentifierType cc = TestUtils.identifierTypeCC();
+        return User.builder().
+                id(1).
+                email("admin@endava.com").
+                phoneNumber("3212312321").
+                identifier("1001000000").
+                firstName("Endava").
+                lastName("Admin").
+                password("$2a$10$bUcuzJbChZheTqPERIqk3u7COWhAF1CV6OU.LUkCG6iZYRhXydRqW").
+                isEmailVerified(true).
+                isApproved(true).
+                role(adminRole).
+                identifierType(cc).
+                bankAccounts(new ArrayList<>()).build();
+    }
+
+    public static UserToApproveAccountDto getUserApprovedAccountDto() {
+        return UserToApproveAccountDto.builder().
+                id(1).
+                email("user@endava.com").
+                isApproved(true).
+                firstName("Endava").
+                lastName("User").build();
+    }
+
+    public static UserToApproveAccountDto getUserNotAprrovedAccountDto() {
+        UserToApproveAccountDto userToApproveAccountDto = getUserApprovedAccountDto();
+        userToApproveAccountDto.setApproved(false);
+        return userToApproveAccountDto;
+    }
+
+    public static User getUserNotAdmin() {
+        Role userRole = TestUtils.userRole();
+        IdentifierType cc = TestUtils.identifierTypeCC();
+        return User.builder().
+                id(1).
+                email("user@endava.com").
```

Comment:
Hi,

This repository is public, so please do not use Endava information or sensitive information.